ngraph/python/test/ngraph/test_create_op.py

```diff
@@ -537,18 +536,6 @@ def test_convert_like():
     assert node.get_output_element_type(0) == Type.i8
 
 
-def test_one_hot():
```

Comment:
Why are you removing these tests?

Reply:
It's duplicated here: https://github.com/arogowie-intel/openvino/blob/aosewski/ng_py_tensor_iterator/ngraph/python/test/ngraph/test_sequence_processing.py#L34-L43